### PR TITLE
Show the git commit version in screenshots/videos [Jak 2]

### PR DESCRIPTION
**Pull request by Zedb0T**
With issues/screenshots becoming more common its probably time to turn this on for debugging reasons.

### CODE REVIEW

**Comment by Zedb0T** [comment]
(Can set *display-sha* to false in the startup.gc file to disable it for main devs)

File: goal_src/jak1/pc/pckernel-h.gc [hunk 1]
```diff
@@ -374,7 +374,6 @@
 (define *display-heap-status* #f)
 (define *display-actor-counts* #f)
 (define *display-text-box* #f)
-(define *display-sha* *debug-segment*)
```
Comment (ManDude) [inline goal_src/jak1/pc/pckernel-h.gc:377]: Why remove this? `pckernel-h` is loaded for all games.

File: goal_src/jak1/pc/pckernel-common.gc [hunk 2]
```diff
@@ -256,7 +258,9 @@
   `(begin
     (cpad-clear! 0 r1)
     (logxor! ,cheats-var (pc-cheats ,cheat))
-    (cheats-sound-play (logtest? ,cheats-var (pc-cheats ,cheat)))
+    (if (-> *pc-settings* speedrunner-mode?)
+    (sound-play "caught-eel")
```
Comment (ManDude) [inline goal_src/jak1/pc/pckernel-common.gc:262]: This sound effect is not in Jak 2! (what's the reason for this change anyway?) Also, fix the indenting please.

**Review by ManDude — CHANGES_REQUESTED** [review]

File: goal_src/jak1/pc/pckernel-common.gc (hunk 2 above)
Comment (Zedb0T) [reply goal_src/jak1/pc/pckernel-common.gc:262]: The reasoning for the change is in the commit 

```Add bad sound feedback if a cheat is attempted to toggle with speedrunner mode on

Before this it just didnt turn on but gave zero feedback so it was kind of bad...```

Then there is a video that is supplied as well. Do you have any suggestions for a sound that is present in both games?

Comment (ManDude) [reply goal_src/jak1/pc/pckernel-common.gc:262]: Oh, well it wasn't apparent that speedrun mode locked out cheats. Why not just *not set* the cheat variable instead of this, then?

Comment (ManDude) [reply goal_src/jak1/pc/pckernel-common.gc]: This code still lets the cheat code go through. If you input the cheat for blue eco, you will still get blue eco even with speedrun mode enabled.
image
